GSA/digitalgov.gov
# Added x logo to sprite.svg

## Summary

This PR will update the twitter text and logo to match the new X format.

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/nl-update-x-logo/)



### Solution

Updated the sprite command in the gulp file along with the styles.js file and also added the x.svg to the sprite.svg file so that it could then render on the page


> [!NOTE]
> Gulp build commands need to be updated, they are currently written but not hooked into the build commands
> Created a card in trello to capture this work

### How To Test

1. Visit [homepage](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/nl-update-x-logo/) and scroll down to the bottom and check that the twitter is now X and link takes user to correct page. 
2. Visit [authors page](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-update-x-logo/authors/18f-team/)
3. Visit [news or events page](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-update-x-logo/event/2023/09/21/uswds-monthly-call-september-2023/) and look at the share tools sidebar

---

### Dev Checklist


tagging supervisor for reference - @rayestrada
